ProjFS.Mac/PrjFSKext/PrjFSKext/VirtualizationRoots.cpp

```diff
@@ -112,9 +174,9 @@ int16_t VirtualizationRoots_LookupVnode(vnode_t vnode, vfs_context_t context)
             RWLock_AcquireExclusive(s_rwLock);
             {
                 // Vnode may already have been inserted as a root in the interim
-                rootIndex = FindRootForVnode_Locked(vnode, vid, fsidInode);
+                rootIndex = FindRootAtVnode_Locked(vnode, vid, fsidInode);
                 
-                if (rootIndex < 0)
+                if (rootIndex == RootHandle_None)
```

Comment:
Similar comment\question on checking None vs < 0 here.  

When should we be checking against None and when should be be checking `VirtualizationRoot_IsValidRootHandle`?

Reply:
Yeah, same answer as above - a result of `RootHandle_ProviderTemporaryDirectory` means we don't need to drop to file I/O level as we've found what we're looking for.